src/ui/FileViewer/hooks/useSwipe.ts

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+
+interface SwipeProps {
+  moveSliderLeft: () => void;
+  moveSliderRight: () => void;
+}
+
+const SWIPE_THRESHOLD_IN_MILLIES = 150;
```

Comment:
it's also good to manage in the one place
https://github.com/sendbird/sendbird-uikit-react/blob/main/src/utils/consts.ts